.github/workflows/at_server_dev_deploy.yaml

```diff
@@ -2,7 +2,7 @@ name: at_root
 on:
   push:
     branches:
-      - trunk
+      - ci-cd-test
```

Comment:
I think trunk still needs to be the branch we use, as `ci-cd-test` will go once this PR is merged.

Reply:
This is not a final request for review, just added the ci-cd-test branch to test it.